docs/sources/mimir/set-up/_index.md

```diff
@@ -0,0 +1,15 @@
+---
+aliases:
+  - operators-guide/deploy-grafana-mimir/
+title: Set up Mimir
+menuTitle: Set up
+description: Learn how to set up a Grafana Mimir server or cluster and visualize data.
+keywords:
+  - Mimir deployment
+  - Mimir Kubernetes
+weight: 15
+---
+
+# Set up Mimir
+
+{{< section menuTitle="true" >}}
```

Comment:
With the current weights this results in:
![image](https://github.com/grafana/mimir/assets/13435893/dd5003fc-842b-4ec7-b71b-d794d28f5538)

I'd put Helm forward as that's the method we advocate, cc @09jvilla

Reply:
Pushed a commit for this.